### DISCUSSION TITLE
be: What is the folder 'data/mini_download' used for?

### DISCUSSION THREAD

**Discussion by bigbang20061** [apache/doris#26962] [A - General / Q&A]
What is the folder 'data/mini_download' used for? Because I found that there are some directories starting with "_ shard" in this directory, and they are all very large in size. I have around 140GB in this directory, and other BE directories do not have this directory

**Comment by tgho-brrrr** [comment]
it is used to store the data to load, and will be deleted after some time (you can configure the time with this https://doris.apache.org/docs/dev/admin-manual/config/be-config/#load_data_reserve_hours)
